docker/Dockerfile

```diff
@@ -5,22 +5,25 @@
 #
 # To run:
 #	docker run -it --name niceman \
-#		-v /path/to/your/local/niceman:/niceman \
+#		-v $PWD:/root/niceman \
```

Comment:
may be here I overdid it a bit ;)  just thought to make it cut/paste-able but anyways due to comments can't be done anyways. so feel free to mandate me to revert or revert yourself